### DISCUSSION TITLE
Single-Note Interaction

### DISCUSSION THREAD

**Discussion by Amras2322** [nicobrauchtgit/obsidian-Smart2Brain#165] [Q&A] [answered]
How can I prompt the SSB to reference a single specific note (or even several notes)? Would be awesome if you could [[Note title]] like how backlinks work, or @ specific folders in your vault.

**Comment by Leo310** [comment, marked as answer]
I know Its been some time, but it will be supported in the next Release ;)
Wont take long anymore!
